contact.html

```diff
@@ -0,0 +1,123 @@
+                zoom: 15,
+                center: quebec,
+            });
+            // The marker, positioned at Quebec
+            const marker = new google.maps.Marker({
+                position: quebec,
+                map: map,
+            });
+        }
+
+        window.initMap = initMap;
+
+    </script>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg bg-body-tertiary">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="index.html">S.A.</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="nav-link" href="index.html">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="about.html">À propos</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="projects.html">Mes projets</a>
+                        </li>
+                        <li class="nav-item active">
+                            <a class="nav-link" aria-current="page" href="contact.html">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <main class="container my-3">
+        <div class="container my-3 text-center">
+            <h1>Contact</h1>
+        </div>
+
+        <div class="container">
+            <div class="row">
+                <div class="col-6">
+                    <form action="">
+                        <div class="form-group">
+                            <label for="courriel">Courriel</label>
+                            <input type="email" class="form-control" id="courriel" placeholder="xyz@xyz.com">
+                        </div>
+                        <div class="form-group">
+                            <label for="sujet">Sujet</label>
+                            <input type="text" class="form-control" id="sujet" placeholder="Sujet">
+                        </div>
+                        <div class="form-group">
+                            <label for="message">Message</label>
+                            <textarea class="form-control" id="message" rows="3"></textarea>
+                        </div>
+                    </form>
+                </div>
+                <div class="col-6">
+                    <div class="row">
```

Comment:
On mobile, you could have let your map in another row to give it more space.

![image](https://user-images.githubusercontent.com/6362750/235277358-1777dcfe-e3c5-482c-8868-a7bd168aaf3d.png)